src/components/capture/create/index.tsx

```diff
@@ -228,128 +191,81 @@ function NewCaptureModal() {
     };
 
     return (
-        <Dialog
-            open
-            onClose={handlers.close}
-            scroll="paper"
-            fullScreen={fullScreen}
-            fullWidth={!fullScreen}
-            maxWidth="md"
-            sx={{
-                '.MuiDialog-container': {
-                    alignItems: 'flex-start',
-                },
-            }}
-            aria-labelledby="new-capture-dialog-title"
-        >
-            <DialogTitle id="new-capture-dialog-title">
-                <FormattedMessage id="captureCreation.heading" />
-                <IconButton
-                    aria-label="close"
-                    onClick={handlers.close}
+        <PageContainer>
+            <Toolbar>
+                <Typography variant="h6" noWrap>
+                    <FormattedMessage id="captureCreation.heading" />
+                </Typography>
+
+                <Stack
+                    direction="row"
+                    alignItems="center"
```

Comment:
Still not 100% sure if this is the best way to make a button group... but it works and is easy. Only weird part is the buttons need to be in reverse order in the JSX.